config.yaml.example

```diff
@@ -16,6 +16,14 @@ integrations:
     config:
       AUTH_TOKEN: example-token
       ORGANIZATION_SLUG: example-organization
+storage:
+  - 
+      engine: memgraph
+      config: 
+        username:
+        password:
+        uri: bolt://localhost:3000
```

Comment:
Is port 3000 listed here the default port for uploading data to Memgraph?  I'm by no means an expert at Memgraph, but I was under the impression that like Neo4j it uses port 7687 for the bolt connection to upload data.

Reply:
You are completely right, Memgraph uses the same port. Memgraph Lab has 3000 for the visualization part so this was a bit of a "tip-feller" from my side